plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -330,20 +331,45 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         return new ConnectorTableProperties();
     }
 
+    @Override
+    public ConnectorTableBasicMetadata getTableBasicMetadata(ConnectorSession session, ConnectorTableHandle table)
+    {
+        JdbcTableHandle handle = (JdbcTableHandle) table;
+
+        return new ConnectorTableBasicMetadata(
+                getSchemaTableName(handle),
+                getColumnMetadata(session, handle).stream()
+                        .map(ColumnMetadata::toBasicMetadata)
```

Comment:
Seems you can first `getColumnMetadata` and later  `.map(ColumnMetadata::toBasicMetadata)` only because `ColumnMetadata` does not have column properties. So actually it masks current inexistent problem with loading column properties for jdbc connectors, but problem would still exist if jdbc connector would have properties.

Reply:
That is true. If we ever add column properties to JDBC connectors I don't believe that `JdbcColumnHandle` will be able to create `ColumnMetadata`. And surely we don't want `io.trino.plugin.jdbc.JdbcClient#getColumns` to query remote data source for column properties.